packages/app-mobile/components/NoteEditor/ImageEditor/autosave.ts

```diff
@@ -0,0 +1,24 @@
+import Setting from '@joplin/lib/models/Setting';
+import { reg } from '@joplin/lib/registry';
+import shim from '@joplin/lib/shim';
+
+export const autosaveFilename = 'autosaved-drawing.joplin.svg';
```

Comment:
Since all drawings have the same filename, could there be issues with one drawing loading another drawing file?